packages/ove-app-networks/src/swagger-extensions.yaml

```diff
@@ -0,0 +1,120 @@
+tags:
+- name: "operation"
+  description: "Operations that can be performed on one or all networks"
+paths:
+  /operation/search:
+    get:
+      tags:
+      - "operation"
+      summary: "Searches for nodes or edges that match provider filters. Filters must be specified according to the OData v3.0 $filter specification."
+      produces:
+      - "application/json"
+      parameters:
+      - name: "filter"
+        in: "query"
+        type: "string"
+        description: "Filter for nodes"
+        required: false
+      - name: "edgeFilter"
+        in: "query"
+        type: "string"
+        description: "Filter for edges"
+        required: false
+      - name: "oveSectionId"
+        in: "query"
+        type: "integer"
+        format: "int32"
+        description: "Section ID"
+        required: false
+      responses:
+        200:
+          description: "successful operation"
+  /operation/color:
+    get:
+      tags:
+      - "operation"
+      summary: "Recolors nodes or edges that match provider filters. Filters must be specified according to the OData v3.0 $filter specification."
+      produces:
+      - "application/json"
+      parameters:
+      - name: "filter"
+        in: "query"
+        type: "string"
+        description: "Filter for nodes"
+        required: false
+      - name: "color"
+        in: "query"
+        type: "string"
+        description: "New color for nodes in the format rgb(x, y, z)"
```

Comment:
An example here would help, why x,y,z rather than r,g,b? int or hex code accepted?

Reply:
I believe that this can be improved to match https://github.com/jacomyal/sigma.js/blob/4ce817bd6f66f24f56a94965afdbebe5226ef9f6/src/utils/sigma.utils.js#L116, which is ulimately what we support. However, the hash cannot be used in a URL, and therefore, we will have to explicitly add that as the first character.

Reply:
James has suggested a fix to this and I have incorporated that in my fixes.